project.clj

```diff
@@ -6,17 +6,31 @@
   :profiles {:dev {:dependencies [[org.clojure/clojure "1.10.0"]
                                   [criterium "0.4.4"]
                                   [cloverage "1.0.13"]
-                                  [fudje "0.9.7"]]
+                                  [fudje "0.9.7"]
+                                  [pjstadig/humane-test-output "0.9.0"]
```

Comment:
This is orthogonal to this PR, please remove.

Reply:
Ok will remove the dev tooling side of things at the end, it is just so useful at the moment 😁

Reply:
Just as a handy tip, you can add these deps to your ~/.lein/profiles.clj profile so you can always have it in arbitrary projects

Reply:
Yeah no I know, I have just opened the conf from `cljs-tooling`. I will delete them, even though they improve the report and workflow in general. I generally dislike having globals in `profile.clj` and prefer to have self-contained projects...not a problem at all here - just explaining the rationale 😁